docs/hacking-guide/en/code-coverage-report.md

```diff
@@ -0,0 +1,37 @@
+# Code coverage report
+
+## Getting JaCoCo exec files
+
+Before you can generate code coverage report by JaCoCo tool,
+you need to get data about what lines of code were executed
```

Comment:
No, coverage reports are not generated always. Later in the paragraph it is written that you have to run tests with `jacoco` maven profile to generate the data. The profile is also used in the example command below the text.